nx-X11/lib/configure.ac

```diff
@@ -115,7 +115,16 @@ fi
 AC_SUBST(XTHREADLIB)
 AC_SUBST(XTHREAD_CFLAGS)
 
-AC_CHECK_FUNC(poll, [AC_DEFINE(USE_POLL, 1, [poll() function is available])], )
+AC_CHECK_FUNC(poll, [has_poll="yes"], [has_poll="no"])
+
+AC_ARG_ENABLE(polling,
```

Comment:
"polling" is not a good name, since polling is still done, albeit via `select()`. Better call it "poll" and describe it as "Disable poll() ...".

Also, we don't need an automake conditional, since we're not using the variable in a Makefile.


```m4
AC_CHECK_FUNC(poll, [has_poll="yes"], [has_poll="no"])

AC_ARG_ENABLE([poll],
               AS_HELP_STRING([--disable-poll],
                 [Disable poll() usage in XlibInt]),
               AS_IF([test "x$enable_poll" = "xno"],
                     [want_poll="no"], [want_poll="yes"]),
               [want_poll="yes"])

if test x"$has_poll" = "xyes" && test x"$want_poll" = "xyes"; then
       use_poll="yes"
       AC_DEFINE(USE_POLL,1,[Enable using the poll() function in XlibInt])
else
       use_poll="no"
fi
```

(May want to adapt whitespace.)